index.html

```diff
@@ -2905,6 +2915,166 @@ <h3>Evidence</h3>
+<code>DIDAuthWithSubjectIdConfirmation</code>) and MAY specify an
+<code>id</code>. The precise <a>properties</a> and semantics of each
+confirmation method are determined by the specific
+<code>confirmationMethod</code> type definition.
+            </p>
+          </dd>
+        </dl>
+
+        <p>
+The type <code>DIDAuthWithSubjectIdConfirmation</code> defines that a
+<a>verifier</a> MAY validate the confirmation method by verifying the proof of
+the <a>verifiable presentation</a> with the verification material of one of the
+authentication verification relationships of the
+<code>credentialSubject.id</code> if the particular
+<code>credentialSubject.id</code> is a Decentralized Identifier (DID).
+        </p>
+
+       <pre class="example nohighlight"
+          title="Usage of the confirmationMethod property of type DIDAuthWithSubjectIdConfirmation">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  "issuer": "https://example.edu/issuers/14",
+  "validFrom": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    <span class="highlight">"confirmationMethod": {
+      "type": "DIDAuthWithSubjectIdConfirmation"
+    }</span>,
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    }
+  },
+  "proof": { <span class="comment">...</span> }
+}
+        </pre>
+
+        <p>
+The type <code>VerificationKeyConfirmation</code> defines that a
+<a>verifier</a> MAY validate the confirmation method by verifying the proof of
+the <a>verifiable presentation</a> with the verification material contained in
+<code>publicKeyJwk</code> or <code>publicKeyMultibase</code>.
+        </p>
+
+       <pre class="example nohighlight"
+          title="Usage of the confirmationMethod property of type VerificationKeyConfirmation">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  "issuer": "https://example.edu/issuers/14",
+  "validFrom": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    <span class="highlight">"confirmationMethod": [{
+      "type": "VerificationKeyConfirmation",
+      "publicKeyJwk": {
+        "crv": "Ed25519",
+        "x": "VCpo2LMLhn6iWku8MKvSLg2ZAoC-nlOyPVQaO3FxVeQ",
+        "kty": "OKP",
+        "kid": "_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A"
+      }
+    },{
+      "type": "VerificationKeyConfirmation",
+      "publicKeyMultibase": "zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+    }]</span>,
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    }
+  },
+  "proof": { <span class="comment">...</span> }
+}
```

Comment:
I like the examples, however I wonder if it is best to include a confirmation method example that does not include key proofs, e.g. a portrait picture or the id number of a passport document, as we have described it in our RWOT#11 paper on identifier bindings. This might help readers understand the concept and open their mind

Reply:
I think this is further evidence that we're talking about something more broad than "confirmation methods", i.e., "assurance" of some kind. It seems that a "confirmation method" could be one such example of an assurance method -- if we even want to use that language at all. But "confirmation method" seems to describe something very specific when used in other places in the ecosystem (i.e., confirmation of the possession of key material -- which, btw, it doesn't even actually confirm).

Reply:
IMO, portrait picture generally makes sense. For example in the ICAO DTC use case, where a traveller registers their DTC before travelling. The DTC might include a confirmation method with a biometric template, e.g., picture. When crossing the border, the automated border control system could then check whether a traveller with that biometric template was registered and let them through.